### PR TITLE
issues/28 and css comunuty

### PR DESCRIPTION
**Pull request by israeldahan**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zohar) [inline .idea/modules.xml:1]: @israel123 why is this file part of your commit?

File: .idea/openideal.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zohar) [inline .idea/openideal.iml:1]: @israel123 why is this file part of your commit?

File: modules/features/fe_idea/fe_idea.features.user_permission.inc [hunk 3]
```diff
@@ -51,6 +51,7 @@ function fe_idea_user_default_permissions() {
   $permissions['create idea content'] = array(
     'name' => 'create idea content',
     'roles' => array(
+      'administrator' => 'administrator',
```
Comment (zohar) [inline modules/features/fe_idea/fe_idea.features.user_permission.inc:54]: `authenticated user` is enough. Why do you add `administrator` as well?

Comment (israeldahan) [reply modules/features/fe_idea/fe_idea.features.user_permission.inc:54]: this is automatic, i'm don't add this.

Comment (zohar) [reply modules/features/fe_idea/fe_idea.features.user_permission.inc:54]: You should go over the diff before you commit and verify it's valid. If needed - edit and recommit

**Pull request closed without merging**